src/pseudopeople/configuration/generator.py

```diff
@@ -83,7 +84,7 @@
 def get_configuration(
     overrides: Optional[Union[Path, str, Dict]] = None,
     dataset_schema: DatasetSchema = None,
-    user_filters: List[Tuple[Union[str, int, pd.Timestamp]]] = None,
+    user_filters: list[tuple[str, str, Union[str, int, pd.Timestamp]]] = None,
```

Comment:
It doesn't complain that this isn't `Optional` or a `Union` with `None` despite having a default `None` value?

Reply:
It's probably a different error type. I added `Optional`

Reply:
hm, it didn't like that `user_filters` could be None later on. maybe just setting the default to `[]` would be better?